src/OpenTelemetry.Exporter.Zipkin/ZipkinExporter.cs

```diff
@@ -90,9 +91,7 @@ private Task SendBatchActivityAsync(IEnumerable<Activity> batchActivity)
                 Content = new JsonContent(this, batchActivity),
             };
 
-            // avoid cancelling here: this is no return point: if we reached this point
```

Comment:
Zipkin previously ignored the cancellationToken it was passed. Spec seems to want us to have a timeout mechanism.